changelog/unreleased/issue-14153.toml

```diff
@@ -0,0 +1,8 @@
+type = "fixed"
+message = "Use PKCS5 instead of ISO10126 padding for secrets encryption"
```

Comment:
This is quite cryptic for regular users. How about something like "Improve JVM security provider compatibility by switching to a widely supported cipher transformation."